components/CreateOrganizationForm.js

```diff
@@ -86,6 +89,9 @@ const CreateOrganizationForm = props => {
     if (values.name.length > 50) {
       errors.name = intl.formatMessage(orgMessages.errorName);
     }
+    if (values.legalName.length > 256) {
```

Comment:
My bad, max length is 255

```suggestion
    if (values.legalName.length > 255) {
```